pgsql/pc_editor.c

```diff
@@ -13,6 +13,11 @@
 #include "pc_pgsql.h"	   /* Common PgSQL support for our type */
 
 Datum pcpatch_rotate_quaternion(PG_FUNCTION_ARGS);
+Datum pcpatch_translate(PG_FUNCTION_ARGS);
+Datum pcpatch_affine(PG_FUNCTION_ARGS);
+Datum pcpoint_rotate_quaternion(PG_FUNCTION_ARGS);
+Datum pcpoint_translate(PG_FUNCTION_ARGS);
+Datum pcpoint_affine(PG_FUNCTION_ARGS);
```

Comment:
Could you please help me reproduce the problem? Do you use autotools or cmake? Do you use specific flags? Thanks.